### PR TITLE
OLS-113: Check for histogram in metrics

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Check for histogram in metrics

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Integration tests

## Related Tickets & Documents

- Related Issue #[OLS-113](https://issues.redhat.com//browse/OLS-113)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: tests/integration/test_metrics.py [hunk 1]
```diff
@@ -90,3 +91,25 @@ def test_rest_api_call_counter_not_found_status():
...
+
+    # first: check summary and statistic
+    assert 'response_duration_seconds_count{path="/metrics/"}' in response_text
+    assert 'response_duration_seconds_sum{path="/metrics/"}' in response_text
+
+    # second: check the histogram itself
+    pattern = re.compile(
+        r"response_duration_seconds_bucket{le=\"0\.[0-9]+\",path=\"\/metrics\/\"}"
+    )
+    # re.findall() returns empty list if not found, and this empty list is treaded as False
```
Comment (onmete) [inline tests/integration/test_metrics.py:110]: *treated

Comment (tisnik) [reply tests/integration/test_metrics.py]: fixed, nice catch @onmete

**Pull request merged**